src/models/multiBarChart.js

```diff
@@ -179,11 +179,6 @@ nv.models.multiBarChart = function() {
             if (showLegend) {
                 legend.width(availableWidth - controlWidth());
 
-                if (multibar.barColor())
-                    data.forEach(function(series,i) {
-                        series.color = d3.rgb('#ccc').darker(i * 1.5).toString();
-                    });
```

Comment:
You are right, it could be quite confusing and the shades of gray make more sense. I've solved this in commits 6031909 and 01b2804 (for the two types of multibar charts). The result is that the default behaviour is per series colouring but in shades of gray (as before). If chart.barColor() is set, the bars are coloured as expected but the legend stays gray. If chart.color() is set, both the legend and the series as colored the same color. If both chart.color() and chart.barColor() are set, the legend follows chart.color colouring and the chart follows chart.barColor colouring. Do you see anything else that's missing?